packages/commands/src/evmDeposit.ts

```
@@ -0,0 +1,125 @@
+  keyRaw: string;
+  rpc: string;
+}) => {
+  try {
+    const chainId = parseInt(options.network);
+    const networkType = getNetworkType(chainId);
+    const rpcUrl = options.rpc || getRpcUrl(chainId);
+    const provider = new ethers.JsonRpcProvider(rpcUrl);
+    let signer;
+    try {
+      signer = new ethers.Wallet(options.keyRaw, provider);
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)